### PR TITLE
Adjust mania scoring to be more in line with ScoreV2 + 85% acc / 15% combo

### PR DESCRIPTION
**Pull request by smoogipoo**
See feedback in https://github.com/ppy/osu/discussions/25810#discussioncomment-7897894 + https://discord.com/channels/188630481301012481/1186947899796500590

15% combo is the lowest I'm willing to go.

This will need the same sort of considerations as https://github.com/ppy/osu/pull/25887#issuecomment-1864074315, eventually...

### CODE REVIEW

File: osu.Game/Database/StandardisedScoreMigrationTools.cs [hunk 1]
```diff
@@ -367,8 +367,8 @@ public static long ConvertFromLegacyTotalScore(ScoreInfo score, LegacyBeatmapCon
 
                 case 3:
                     return (long)Math.Round((
-                        990000 * comboProportion
-                        + 10000 * Math.Pow(score.Accuracy, 2 + 2 * score.Accuracy)
+                        850000 * comboProportion
+                        + 150000 * Math.Pow(score.Accuracy, 2 + 2 * score.Accuracy)
```
Comment (bdach) [inline osu.Game/Database/StandardisedScoreMigrationTools.cs:371]: Given that this is in `ConvertFromLegacyTotalScore()`, should we be bumping score version again to upgrade previously-imported stable scores to the new proportions?

(Key point, I am not talking about *lazer* scores here; emphasis on legacy total score.)

Comment (peppy) [reply osu.Game/Database/StandardisedScoreMigrationTools.cs:371]: Might as well, yeah.

**Comment by bdach** [comment]
Also, to be doubly clear: if we merge this in its current state, we will have to reopen https://github.com/ppy/osu/issues/24238. Moving extra score for PERFECTs to bonus was supposed to make SSes achievable on scores that do not have all PERFECTs.

Is this OK?

**Comment by peppy** [comment]
Time and game mechanics are apparently a circle.

I'm honestly not sure which is better, but we're going to merge this and find out (hopefully sooner rather than later) which voice is stronger.

**Review by bdach — APPROVED** [review]

**Pull request merged**